pt-br/pages/conta-bancaria.md

```diff
@@ -16,7 +16,12 @@ Clientes do DB podem realizar saques (sem custo) em caixas eletrônicos dos segu
 
 ## N26
 
-Abrir uma conta no N26 é relativamente fácil e rápido. Cadastre-se em [https://n26.com/](https://n26.com/) e siga os passos indicados. Será necessário enviar um scan de seu passaporte e parear seu smartphone com o aplicativo (um número de telefone da Alemanha é requerido). Após, será necessário verificar sua identidade para que sua conta fique ativa. Há duas formas de fazer essa verificação:
+Abrir uma conta no N26 é relativamente fácil e rápido. Cadastre-se em [https://n26.com/](https://n26.com/) e siga os passos indicados. Será necessário enviar um scan de seu passaporte, registro de residência e parear seu smartphone com o aplicativo (um número de telefone da Alemanha é requerido). Após, será necessário verificar sua identidade para que sua conta fique ativa. Há duas formas de fazer essa verificação:
```

Comment:
Agora precisamos de resident permit
https://support.n26.com/en-eu/getting-started/personal-information/how-to-prove-my-identity